src/main/kotlin/game/racing/RacingGameApplication.kt

```diff
@@ -1,12 +1,17 @@
 package game.racing
 
+import game.racing.controller.RacingGameController
 import game.racing.domain.CarFactory
 import game.racing.domain.RacingGame
 import game.racing.domain.RacingManager
 import game.racing.view.InputView
 
 fun main() {
     val cars = CarFactory.generateCarByCarNames(InputView.inputCarNames())
-    val racingGame = RacingGame(RacingManager(cars), InputView.inputTryNumber())
-    racingGame.start()
+    val racingManager = RacingManager(cars)
+    val racingGame = RacingGame(racingManager)
+
+    RacingGameController.initializeGame(racingManager.cars)
+    val resultList = RacingGameController.progressGame(racingGame, InputView.inputTryNumber())
+    RacingGameController.showGameResult(resultList, racingManager.determineWinner())
```

Comment:
아마 저였으면 `RacingGameApplication`을 Controller로 생각하고 
main  함수에서 아래와 같이 전개를 했을 것 같아요. 🤔

```suggestion
    val cars = CarFactory.generateCarByCarNames(InputView.inputCarNames())
    val racingManager = RacingManager(cars)
    val racingGame = RacingGame(racingManager)

    val resultList = racingGame.tryMovesRepeatedly(InputView.inputTryNumber())
    resultList.forEach(ResultView::printResult)
    ResultView.printWinner(racingManager.determineWinner())
```

Reply:
아 그러네요, 불필요하게 main과 controller가 비슷한 역할을 갖고 있었네요.. 수정하겠습니다.